cookbooks/aws-parallelcluster-environment/files/cloudwatch/write_cloudwatch_agent_json.py

```diff
@@ -61,9 +64,21 @@ def add_instance_log_stream_prefixes(configs):
     return configs
 
 
+def render_jinja_template(template_file_path, **kwargs):
```

Comment:
Adding this as part of another python module like `cloudwatch_agent_common_utils.py` whihc can be used by both the other modules would be more beneficial.

So is addition of the `default_platforms` in one single Place as we would need to change this again in two places.